modules/ROOT/pages/configure-service-mesh.adoc

```diff
@@ -121,20 +125,21 @@ image::create-non-mule-api-service-mesh.png[85%, 85%]
 +
 Your API is now created.
 
+[[bind-adapter]]
 == Bind your service to an API on API Manager
```

Comment:
What is "your service"? The user created an API and provisioned an adapter, but a service...?

If you keep this title:
```suggestion
== Bind Your Service to an API on API Manager
```